android/src/main/java/nl/fhict/happynews/android/PostManager.java

```diff
@@ -25,18 +28,20 @@ public static PostManager getInstance() {
         return ourInstance;
     }
 
-    private String API_URL = "https://happynews-api.svendubbeld.nl/post";
-    private ArrayList<Post> newPosts;
+    private String API_URL = "https://happynews-api.svendubbeld.nl";
     private FeedAdapter feedAdapter;
 
+    private MainActivity activity;
```

Comment:
I think we should write a listener interface for this, passing a specific activity here is not very extensible.